jobs/build/build-sync/Jenkinsfile

```diff
@@ -110,89 +109,152 @@ node {
     ])  // Please update README.md if modifying parameter names or semantics
 
     commonlib.checkMock()
-    echo("Initializing ${params.BUILD_VERSION} sync: #${currentBuild.number}")
-    build.initialize()
+
+    stage("Initialize") {
+        echo("Initializing ${params.BUILD_VERSION} sync: #${currentBuild.number}")
+        currentBuild.displayName = "${BUILD_VERSION} - ${ASSEMBLY}"
+
+        // doozer_working must be in WORKSPACE in order to have artifacts archived
+        mirrorWorking = "${env.WORKSPACE}/MIRROR_working"
+        buildlib.cleanWorkdir(mirrorWorking)
+
+        if (params.DRY_RUN) {
+            currentBuild.displayName += " [DRY_RUN]"
+        }
+
+        def arches = buildlib.branch_arches("openshift-${params.BUILD_VERSION}").toList()
+        if ( params.EXCLUDE_ARCHES ) {
+            excludeArches = commonlib.parseList(params.EXCLUDE_ARCHES)
+            currentBuild.displayName += " [EXCLUDE ${excludeArches.join(', ')}]"
+            if ( !arches.containsAll(excludeArches) )
+                error("Trying to exclude arch ${excludeArches} not present in known arches ${arches}")
+            arches.removeAll(excludeArches)
+        }
+        currentBuild.description = "Arches: ${arches.join(', ')}"
+
+        if ( params.DEBUG ) {
+            logLevel = " --loglevel=5 "
+        }
+
+        imageList = commonlib.cleanCommaList(params.IMAGES)
+        if ( imageList ) {
+            echo("Only syncing specified images: ${imageList}")
+            currentBuild.description += "<br>Images: ${imageList}"
+        }
+
+        failCountFile = "${BUILD_VERSION}-assembly.${ASSEMBLY}.count"
+    }
 
     stage("Version dumps") {
         buildlib.doozer "--version"
         echo "${buildlib.DOOZER_BIN}"
         sh "oc version -o yaml"
     }
 
-    failCountFile = "${BUILD_VERSION}-assembly.${ASSEMBLY}.count"
-    currentBuild.displayName = "${BUILD_VERSION} - ${ASSEMBLY}"
-
-    try {
-
+    stage ("build sync") {
+        sh "rm -rf ./artcd_working && mkdir -p ./artcd_working"
+        def cmd = [
+            "artcd",
+            "-v",
+            "--working-dir=./artcd_working",
+            "--config=./config/artcd.toml",
+        ]
+        if (params.DRY_RUN) {
+            cmd << "--dry-run"
+        }
+        cmd += [
+            "build-sync",
+            "--version=${params.BUILD_VERSION}",
+            "--assembly=${params.ASSEMBLY}"
+        ]
+        if (params.PUBLISH) {
+            cmd << "--publish"
+        }
+        cmd << "--data-path=${params.DOOZER_DATA_PATH}"
+        if (params.EMERGENCY_IGNORE_ISSUES) {
+            cmd << "--emergency-ignore-issues"
+        }
+        if (params.EMERGENCY_IGNORE_ISSUES) {
+            cmd << "--emergency-ignore-issues"
+        }
```

Comment:
duplicate

Reply:
Fixed in 64a81865f595777aceab58e32e7e4ba759289b0e